modules/core/src/main/java/org/apache/ignite/internal/managers/discovery/GridDiscoveryManager.java

```diff
@@ -836,7 +856,8 @@ else if (type == EVT_CLIENT_NODE_RECONNECTED) {
                                         discoCache0,
                                         notification.getTopSnapshot(),
                                         null,
-                                        notification.getSpanContainer()
+                                        notification.getSpanContainer(),
+                                        null
```

Comment:
Why null here?

Reply:
changed to getting of current subject id